### PR TITLE
fixes door accesses on atlas and destiny

### PR DESCRIPTION
**Pull request by nefarious6th**
## About the PR 
fixes the janitor supply closet to be restricted to janitors only on destiny
fixes artlab to be restricted to scientists only on atlas

## Why's this needed? 
bug bad

### CODE REVIEW

File: maps/destiny.dmm [hunk 1]
```diff
@@ -24831,7 +24831,8 @@
 	icon_state = "4-8"
 	},
 /obj/machinery/door/airlock/gannets/maintenance{
-	name = "airlock-'Crew Lounge'"
+	name = "airlock-'Janitor's Supply Closet'";
+	req_access_txt = "26"
```
Comment (mordent-goonstation) [inline maps/destiny.dmm:24835]: Sanity check on setting this value vs. using an access spawner?

File: maps/destiny.dmm [hunk 2]
```diff
@@ -42583,12 +42584,13 @@
...
 /obj/machinery/door/firedoor/border_only,
-/obj/machinery/door/airlock/gannets{
-	name = "airlock-'Crew Lounge'"
-	},
 /obj/cable{
 	icon_state = "4-8"
 	},
+/obj/machinery/door/airlock/gannets{
+	name = "airlock-'Janitor Supply Closet";
+	req_access_txt = "26"
```
Comment (mordent-goonstation) [inline maps/destiny.dmm:42592]: Sanity check on setting this value vs. using an access spawner?

File: maps/atlas.dmm [hunk 3]
```diff
@@ -12266,7 +12266,7 @@
 "DR" = (
 /obj/machinery/door/airlock/pyro/glass{
 	name = "Research Wing";
-	req_access_txt = "5"
+	req_access_txt = null
```
Comment (mordent-goonstation) [inline maps/atlas.dmm:12269]: Remove this rather than null it, surely?

File: maps/destiny.dmm (hunk 2 above)
Comment (nefarious6th) [reply maps/destiny.dmm:42592]: so, almost none of destiny actually uses access spawners. there's a single one on an airlock into science that has had some issues with maint/sci shared access stuff as a part of an earlier bug fix. i'm not looking to convert destiny to access spawners at this time and intend to do that down road; preferably also around the time or as a part of a few prs for when/if i go in to replace all the doors for perspective ones.

Comment (mordent-goonstation) [reply maps/destiny.dmm:42592]: Not going to verify as I don't do mapping, but I'm fine with consistency within the map. If the rest of this map uses var setting then I'm okay with it here.

**Review by mordent-goonstation — APPROVED** [review]

**Pull request merged**